src/main/java/frc/robot/RobotContainer.java

```diff
@@ -141,8 +141,13 @@ private void configureBindings() {
 
     // Shooter commands
     shooterSubsystem.setDefaultCommand(shooterSubsystem.shooterIdle());
-    OI.getTrigger(OI.Operator.shooterTrigger).onTrue(shooterSubsystem.shooterFire());
-
+    OI.getTrigger(OI.Operator.shooterTrigger)
+        .onTrue(
+            shooterSubsystem
+                .shooterFire()
+                .until(shooterSubsystem.shooterReady())
+                .andThen(triggerSubsystem.getShootCommand())
+                .onlyWhile(OI.getTrigger(OI.Operator.shooterTrigger)));
```

Comment:
We should think about this some more. I think we want the functionality to be like this:
"While trigger is held, spin up shooter to desired RPM. If shooter is ready when trigger is released, shoot the note." 
Currently this code says "shoot as soon as shooter is ready", which isn't quite what we want, I don't think.